src/net/sf/xframe/xsddoc/util/XMLUtil.java

```diff
@@ -24,8 +24,8 @@
 
 import javax.xml.transform.TransformerException;
 
-import org.apache.xml.utils.URI;
-import org.apache.xml.utils.URI.MalformedURIException;
+import org.apache.axis2.databinding.types.URI;
```

Comment:
It seems like you're missing xalan2 on the classpath. 
This exception is part of xalan2 api.

Reply:
Yup, fixed now. :-)